src/pages/atomicdex/api/v20-dev/zhtlc_coins/index.mdx

```diff
@@ -85,26 +65,116 @@ To withdraw ZHTLC coins, you need to use the [task::withdraw](/atomicdex/api/v20
       }
     }
   }
-  ```
+}
+```
 </CodeGroup>
 
-<CollapsibleSection expandedText="Hide Response" collapsedText="Show Response">
-  #### Response
+#### Sync from block 2528700, with custom `.zcash-params` path and scan params
+
+<CodeGroup title="task::enable_z_coin::init" tag="POST" label="task::enable_z_coin::init" mm2MethodDecorate="true">
+```json
+{
+  "userpass": "MM2_RPC_PASSWORD",
+  "method": "task::enable_z_coin::init",
+  "mmrpc": "2.0",
+  "params": {
+    "ticker": "ZOMBIE",
+    "activation_params": {
+      "mode": {
+        "rpc": "Light",
+        "rpc_data": {
+          "electrum_servers": [{"url":"zombie.dragonhound.info:10033"}],
+          "light_wallet_d_servers": ["http://zombie.dragonhound.info:443"],
+          "sync_params": {"height": 2528700}
+        }
+      },
+      "zcash_params_path": "/home/username/path_to/.zcash-params",
+      "scan_blocks_per_iteration": 100,
+      "scan_interval_ms": 200
+    }
+  }
+}
+```
+</CodeGroup>
 
-  ```json
-  {
-    "mmrpc": "2.0",
-    "result": {
-      "task_id": 0
-    },
-    "id": null
+#### Sync from sapling activation height (earliest)
+
```

Comment:
we should explain why sapling activation height is the earliest.

Reason: `ZCoin lightwallet client supports only blocks that are post sapling.`

Reply:
updated, thanks